### PR TITLE
Added two ENVs in webui.py to make it possbile to automate building d…

### PR DESCRIPTION
**Pull request by mongolu**
…ocker image.

In Dockerfile i run start_linux.sh" which in the end it start python webui.py.
In webui.py, at the end, it calls launch_webui().
I included an ENV var called OOBABOOGA_LAUNCH_AFTER_INSTALL which finish execution when false.

The second ENV var is for declaring which GPU choice to use in installation.
It's called OOBABOOGA_INSTALL_GPU_CHOICE, and if it's set beforehand, it's used similarly with current variant of user input.

### CODE REVIEW

File: webui.py [hunk 1]
```diff
@@ -80,15 +80,19 @@ def clear_cache():
...
-    # Select your GPU or, choose to run in CPU mode
-    print("What is your GPU")
-    print()
-    print("A) NVIDIA")
-    print("B) AMD (Linux/MacOS only. Requires ROCm SDK 5.4.2/5.4.3 on Linux)")
-    print("C) Apple M Series")
-    print("D) None (I want to run models in CPU mode)")
-    print()
-    gpuchoice = input("Input> ").lower()
+    if "OOBABOOGA_INSTALL_GPU_CHOICE" in os.environ:
```
Comment (mongolu) [inline webui.py:83]: automate install by using ENV var

File: webui.py [hunk 2]
```diff
@@ -310,6 +314,11 @@ def launch_webui():
             install_dependencies()
             os.chdir(script_dir)
 
+        if "OOBABOOGA_LAUNCH_AFTER_INSTALL" in os.environ:
```
Comment (mongolu) [inline webui.py:317]: Don't launch after install (useful when being executed from Dockerfile)

**Comment by mongolu** [comment]
Please do accept this PR.
I'm using it in a Dockerfile which is relaying on one-click-installer and in the docker-compose it mustn't be running on call.

**Comment by oobabooga** [comment]
Docker is beyond the scope of this installer.

**Comment by mongolu** [comment]
Adding these 2 ENVs doesn't change nothing in the execution logic if they don't exist.
So, kindly please, give it a second look.

**Comment by Josh-XT** [comment]
@oobabooga can you revisit this please? This would be very helpful.

**Pull request merged**